### PR TITLE
1.1 stable

### PR DESCRIPTION
**Pull request by michaeldayreads**
CI cherry picks from master.

### CODE REVIEW

File: docs/RELEASE-NOTES.rst [hunk 1]
```diff
@@ -1,33 +1,34 @@
...
-^^^^^^^^^^^^^^^^^^^
+```````````````````
 
-* Virtual Servers can now be configured on the BIG-IP from Kubernetes Ingress resources.
-* Multiple SSL Profiles can be configured for a Virtual Server.
-* Watches all Kubernetes namespaces by default, or can watch a list of namespaces, or namespaces with a customer specified label. This addresses a previous limitation in v1.0.0.
-* Watches for Kubernetes annotation if virtual address not specified, enabling custom IPAM integration.
-* Creates detached pools if virtual server bind addresses not specified.
+* Creation of BIG-IP Virtual Servers from Kubernetes Ingress resources.
+* Configure multiple SSL Profiles for a BIG-IP Virtual Servers.
```
Comment (amudukutore) [inline docs/RELEASE-NOTES.rst]: Given that we say "for a", I would expect this to say "Virtual Server" and not "Virtual Servers"

Comment (michaeldayreads) [inline docs/RELEASE-NOTES.rst]: @amudukutore - as this is a set of cherry picks, should we open an issue?

Comment (jputrino) [inline docs/RELEASE-NOTES.rst]: @michaeldayreads - seconded. That is a separate issue.

Comment (amudukutore) [inline docs/RELEASE-NOTES.rst]: Yes - separate issue that needs to be fixed in master and cherry-picked into 1.1-stable.

**Pull request merged**